recipe/session/claims/booleanClaim.go

```diff
@@ -0,0 +1,25 @@
+package claims
+
+func BooleanClaim(key string, fetchValue FetchValueFunc, defaultMaxAgeInSeconds *int64) (*TypeSessionClaim, BooleanClaimValidators) {
```

Comment:
why does this function return the claim and the validators? And not just the claim?